documentation/slate/source/includes/realtime.md

```diff
@@ -0,0 +1,263 @@
+
+### Journeys
+
+If the station deleted is the destination of the journey, Navitia will compute an other itinerary to the requested station. If not, the disruption will be present at the root level of the response for information, but it won't be affecting the journey.
+
+### Departures & Stop Schedules
+
+The departure time of the train with the reduced service simply won't be displayed in the list of departures/stop_schedules if "data_freshness" is set to "realtime".
+
+## <a name="MODIFIED_SERVICE"></a>Modified service
+
+![image](modified_service.png)
+
+``` shell
+# Extract of an impacted stop of /disruptions
+{
+    "amended_arrival_time": "185500",
+    "amended_departure_time": "190000",
+    "arrival_status": "added"
+    "cause": "Ajout d'une desserte",
+    "departure_status": "added",
+    "is_detour": false,
+    "stop_point": ⊕{7 items},
+    "stop_time_effect": "added",
+},
+```
+The effect of the disruption is `MODIFIED_SERVICE`. It means that there is one or several stop points added into the journey. This can be at any position in the journey (origin and destination included).
+
+In the disruption, new stop points can be found in the list of "impacted_stops" with the departure/arrival status set to "added". See the [disruption](#disruption) objects section for its full content and description.
+
+<div></div>
+### Journeys
+
+``` shell
+# Request example for /journeys
+http://api.navitia.io/v1/coverage/<coverage>/journeys?from=<origin>&to=<destination>&data_freshness=realtime
+```
+
+``` shell
+# Extract of the public transport section of the response /journeys
+
+    "sections": ⊖[
+       ⊖{
+            "additional_informations": ⊕[1 item],
+            "arrival_date_time": "20190605T204500",
+            "base_arrival_date_time": "20190605T204500",
+            "co2_emission": ⊕{2 items},
+            "data_freshness": "realtime",
+            "departure_date_time": "20190605T160000",
+            "display_informations": ⊕{13 items},
+            "duration": 28560,
+            "from": ⊕{5 items},
+            "geojson": ⊕{3 items},
+            "id": "section_1_0",
+            "links": ⊕[6 items],
+            "stop_date_times": ⊖[
+                ⊕{7 items},
+                ⊖{
+                     "additional_informations": [],
+                     "arrival_date_time": "20190605T185500",
+                     "departure_date_time": "20190605T190000",
+                     "links": []
+                     "stop_point": ⊕{7 items},
+                 },
+                ⊕{7 items},
+            ]
+            "to": ⊕{5 items},
+            "type": "public_transport",
+        },
+    ]
+```
+
+The status of the journey is `MODIFIED_SERVICE`. In a public transport section of the response, "arrival_date_time"/"departure_date_time" are the arrival/departure time of a new stop_point. New stop points are only available when the "data_freshness" parameter is set to "realtime". 
```

Comment:
Same here with "stop_point" ?

Reply:
Ah! Yeah, would be nice to be consistent :)